### PR TITLE
Feat: add button on domains page

### LINKED ISSUE #198: Feat: Add create new domain button on Domains page
image

### PR DESCRIPTION
**Pull request by Myrfion**
Add button on the domains page that gets the user to new domain screen

Closes #198

### CODE REVIEW

File: app/routes/__index/domains/index.tsx [hunk 1]
```diff
@@ -31,6 +32,11 @@ export default function DomainsIndexRoute() {
         been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
         galley of type and scrambled it to make a type specimen book.
       </Text>
+      <Flex justifyContent="flex-end">
+        <Link to="/domains/new">
+          <Button rightIcon={<AddIcon boxSize={3} />}>Create new domain</Button>
```
Comment (humphd) [inline app/routes/__index/domains/index.tsx:37]: I wonder if `Domain` (capital) is better?  Not sure.

**Review by humphd — APPROVED** [review]

**Review by Ririio — APPROVED** [review]
Looks great to me

File: app/routes/__index/domains/index.tsx (hunk 1 above)
Comment (Myrfion) [reply app/routes/__index/domains/index.tsx:37]: image

Comment (Myrfion) [reply app/routes/__index/domains/index.tsx:37]: I don't have a hard opinion on that, if you guts want I can change it to capital

Comment (humphd) [reply app/routes/__index/domains/index.tsx:37]: Same, we should just set a standard and stick with it.

**Pull request merged**